### ISSUE TITLE
Other networks

### ISSUE THREAD

**Issue by Epi-NMT** [liyiying/Feature_Critic#4]
Can this code apply to other pretrained networks? like resnet18, resnet50.
Based on the line 361 in model_PACS, it seems it cannot apply on resnet18.
I have tried resnet50, but there are many details need to be modified.

**Comment by liyiying** [comment]
Yes, it can be applied to other networks. Line 361 here is not to emphasize the version of Resnet "18". The batchnorm function is used in the BasicBlock/Bottleneck which are used by different versions of ResNet. If you read the source code of ResNet carefully, you won't have such questions. Well, actually you need to modify some details if you use other networks. I believe you can find these places to modify by debugging.

**Comment by Epi-NMT** [comment]
Have you ever try your method on resnet18 or resnet50? 
How is the performance?

**Comment by liyiying** [comment]
I have tried some different versions of ResNet on VD instead of PACS, and see some slight improvement with more layers of  ResNet.
